sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/models/InvalidParticipantException.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.chat.models;
+
+import com.azure.android.core.exception.AzureException;
+
+/**
+ * Exception when a participant cannot be added to a chat thread
+ */
+public final class InvalidParticipantException extends AzureException {
```

Comment:
This will make SDK users error handling more complicated since now we have two base classes for excpetions: HttpResponseException and AzureException. We should use only one not both.

Reply:
Based on offline discussion, we would keep InvalidParticipantException extends AzureException